File: DSA github/C-language/linear_search.c
#include<stdio.h>
int linear_search(int a[],int n,int key);
void main()
{
	int i,n;
	int a[100];
	int key,res;
	printf("Enter the no of elements\n");
	scanf("%d\n",&n);
	for(i=0;i<n;i++)
	{
		scanf("%d",&a[i]);
	}
	printf("Enter the elements to be searched: ");
	scanf("%d",&key);
	res=linear_search(a,n,key);
	if(res==-1)
	{
		printf("Element not found");
	}
	else
	{
		printf("Element found at %d position",res+1);
	}
}

int linear_search(int a[],int n,int key)
{
	int i;
	for(i=0;i<=n;i++)
	{
		if(a[i]==key)
		return i;
	}
}

File: DSA github/Python/linearsearch.py
def linear_search(arr,key):
    for i in range(len(arr)):
        if arr[i]==key:
            return i
    return -1
print(" ")
print("Linear Search")
key = int(input("enter element to be searched: "))
size = int(input("enter size of list: "))
arr = []
i = 0
for i in range(size):
    val = int(input("enter the value: "))
    arr.append(val)
x=linear_search(arr,key)
if x != -1:
	print(f"Element is found at {x+1} index position")
else:
	print("Element is not present in array")